app/controllers/better_together/conversations_controller.rb

```diff
@@ -48,11 +49,46 @@ def create # rubocop:todo Metrics/MethodLength, Metrics/AbcSize
     end
 
     def update
+      authorize @conversation
       ActiveRecord::Base.transaction do
         if @conversation.update(conversation_params)
-          redirect_to @conversation
+          @messages = @conversation.messages.with_all_rich_text.includes(sender: [:string_translations]).order(:created_at)
+          @message = @conversation.messages.build
+
+          turbo_stream_response = lambda do
+            if @conversation.conversation_participants.include?(helpers.current_person)
```

Comment:
When possible, prefer to work with a list of IDs over objects to avoid loading extra data from the db.

This can be rewritten (I think) as `@conversation.participant_ids.include?(helpers.current_person.id)`.

In this case, I think that `conversation_participants` return instances of `ConversationParticipant` and you're patching on an instance of `Person`. From a conversation, `participants` returns a collection of Person records.

Also, you'll want to pull this expression out into a variable and use it in both instances, as it's a long duplication of this line on line 71.